python/interpret_community/explanation/explanation.py

```diff
@@ -1633,10 +1633,8 @@ def _get_raw_explainer_create_explanation_kwargs(*, kwargs=None, explanation=Non
     if explanation is not None and kwargs is not None:
         raise ValueError("Both explanation and kwargs cannot be set")
 
-    keys = [ExplainParams.METHOD, ExplainParams.CLASSES, ExplainParams.MODEL_TASK,
-            ExplainParams.CLASSIFICATION, ExplainParams.INIT_DATA, ExplainParams.EVAL_DATA,
-            ExplainParams.EXPECTED_VALUES, ExplainParams.MODEL_ID, ExplainParams.EVAL_Y_PRED,
```

Comment:
shouldn't ExplainParams.EVAL_Y_PRED and ExplainParams.EVAL_Y_PRED_PROBA be the same regardless?